main.nf

```diff
@@ -218,24 +301,129 @@ process get_software_versions {
     fastp -v 2> v_fastp.txt
     megahit --version > v_megahit.txt
     metabat2 -h 2> v_metabat.txt || true
+    NanoPlot --version > v_nanoplot.txt
+    filtlong --version > v_filtlong.txt
+    porechop --version > v_porechop.txt
+    NanoLyse --version > v_nanolyse.txt
+    spades.py --version > v_spades.txt
+    run_BUSCO.py --version > v_busco.txt
 
-    # fake checkm data setRoot so we can read checkm version number
-    chd=\$(readlink -f checkm_data)
-    printf "\$chd\\n\$chd\\n" | checkm data setRoot
-
-    checkm -h > v_checkm.txt
-    refinem -h > v_refinem.txt
     scrape_software_versions.py > software_versions_mqc.yaml
     """
 }
 
 
+/*
+ * Trim adapter sequences on long read nanopore files
+ */
+process porechop { 
+    tag "$id"
+        
```

Comment:
porechop should be optional in my opinion, as the most recent ONT basecaller (guppy) now also removes adapter sequences.

Reply:
I'll do that